### PR TITLE
Reduce Cognitive Complexity as suggested by SonarQube

### PR DESCRIPTION
**Pull request by shwetaap**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: hack/csv-generator.go [hunk 1]
```diff
@@ -166,23 +166,18 @@ func replaceVariables(flags generatorFlags, csv *csvv1.ClusterServiceVersion) er
 			updatedContainer.Image = flags.operatorImage
 			updatedVariables := make([]v1.EnvVar, 0)
 			for _, envVariable := range container.Env {
-
-				if envVariable.Name == common.KvmInfoNfdPluginImageKey {
+				switch envVariable.Name {
```
Comment (akrejcir) [inline hack/csv-generator.go:169]: Can you extract this `switch` block into a function?

**Review by akrejcir — CHANGES_REQUESTED** [review]
Can you also update the description of this PR and remove all unneeded text?

**Comment by ksimon1** [comment]
@shwetaap can you please address Andrej's feedback?

**Comment by akrejcir** [comment]
@shwetaap , can you update this PR? It will only take a few minutes and then we can merge it.

**Comment by dominikholler** [comment]
@vatsalparekh is this PR fixing one of the code smells assigned to you?

**Comment by vatsalparekh** [comment]
@dominikholler Yes, I'll include this in a new PR

**Pull request closed without merging**